contentcuration/contentcuration/assets/shared/views/AppBar.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <VToolbar app dark color="purple" :tabs="Boolean($slots.tabs)">
```

Comment:
One thing to note is that vuetify doesn't handle responsiveness as well as I would like- its default behavior is to hide the side actions completely, so you may need to add some workarounds here

![toolbar](https://user-images.githubusercontent.com/7447496/61415692-4eb47e80-a8a6-11e9-8869-4afd634387ba.gif)

Reply:
Note that the top example has the class `hidden-sm-and-down` applied to it, so this is intended behaviour for this instance. Other examples on the page exhibit better behaviour (including hiding the title rather than the buttons if needed).